Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -29,9 +56,13 @@ public class TextToSpeechManager : MonoBehaviour
         [Tooltip("The audio source where speech will be played.")]
         public AudioSource audioSource;
 
+        [Tooltip("The voice that will be used to generate speech.")]
+        public TextToSpeechVoice voice;
```

Comment:
Do others agree with this? The fields have to be serializable or they can't be set in the inspector, but I'm ok with making them private and exposing properties. And again, just want to make sure we're OK with a potentially breaking change. I am if you guys are.